### PR TITLE
[IMP] theme_*: adapt `s_numbers_framed` across design themes

### PR DESCRIPTION
**Pull request by chgo-odoo**
*: artists, avantgarde, aviato, beauty, bewise, bistro, bookstore, buzzy, clean, cobalt, enark, graphene, kea, kiddo, loftspace, monglia, nano, botes, odoo_experts, orchid, paptic, real_estate, treehouse, vehicle, yes,zap.

This PR adapts the design of the `s_numbers_framed` occurrences across design themes.

- requires https://github.com/odoo/odoo/pull/175700

task-4094384
Part of task-4077427

### CODE REVIEW

File: theme_beauty/views/snippets/s_numbers_framed.xml
Comment (chrisdt1998) [inline theme_beauty/views/snippets/s_numbers_framed.xml:1]: There's something weird happening with this theme. When you select the block, it has white numbers but then it flickers to black once it's been chosen.
image
image

File: theme_bewise/views/customizations.xml
Comment (chrisdt1998) [inline theme_bewise/views/customizations.xml:1]: Same thing here
image
image

File: theme_loftspace/views/snippets/s_numbers_framed.xml
Comment (chrisdt1998) [inline theme_loftspace/views/snippets/s_numbers_framed.xml:1]: Same here:
image

image

File: theme_monglia/views/customizations.xml
Comment (chrisdt1998) [inline theme_monglia/views/customizations.xml:1]: Same here:
image
image

File: theme_notes/views/snippets/s_numbers_framed.xml
Comment (chrisdt1998) [inline theme_notes/views/snippets/s_numbers_framed.xml:1]: Same here:
image
image

**Review by chrisdt1998 — COMMENTED** [review]
There are some issues with some of the blocks. Do you know why this happens?

**Comment by chgo-odoo** [comment]
Thanks @chrisdt1998 👍 

It's due to a missing `o_colored_level` class on the section which is mandatory when we nest `o_cc` (which we do in some themes). It should be fixed now

**Comment by chrisdt1998** [comment]
@chgo-odoo Ah I see. However, I think it would make more sense to instead add o_colored_level to the base snippet section element instead of doing it for each theme. Also, the reason for the 'flicker' is because  
image
adds the o_colored_level class to the section element. That's why in the default snippet, even though you don't explicitly add the o_colored_level class, it will be added there when you drag and drop the snippet.

TLDR: please add o_colored_level to the section element of the default snippet and adjust appropriately for the theme snippets.
